spec/models/customer_spec.rb

```diff
@@ -69,4 +69,41 @@
       expect(c.user).to eq user2
     end
   end
+
+  describe 'scopes' do
+    context 'managed_by' do
```

Comment:
We are missing the test case of the unpersisted user.

Reply:
done